src/main/java/com/ericsson/ei/subscriptionhandler/RunSubscription.java

```diff
@@ -107,19 +107,25 @@ public boolean runSubscriptionOnObject(String aggregatedObject, Iterator<JsonNod
             while (conditionIterator.hasNext()) {
                 String rule = conditionIterator.next().get("jmespath").toString().replaceAll("^\"|\"$", "");
                 String new_Rule = rule.replace("'", "\"");
-                LOGGER.info("Rule : " + rule);
-                LOGGER.info("New Rule after replacing single quote : " + new_Rule);
+                LOGGER.debug("Rule : " + rule);
+                LOGGER.debug("New Rule after replacing single quote : " + new_Rule);
                 JsonNode result = jmespath.runRuleOnEvent(rule, aggregatedObject);
-                LOGGER.info("Result : " + result.toString());
-                if (result.toString() != null && result.toString() != "false" && !result.toString().equals("[]")) {
+                LOGGER.debug("Result : " + result.toString());
```

Comment:
Is these Logger debug message just for testing? If you want to keep those,, write them better and more detailed in fewer logger.debug(msg) calls.

Reply:
Done